spec/cypress/e2e/courses_spec.cy.js

```diff
@@ -74,8 +74,8 @@ describe("Courses", function () {
                     'user_id':1
                 }]
             ]).then((records) => {
-                cy.server();
-                cy.route('**/new').as('new');
+                // cy.server();
+                // cy.route('**/new').as('new');
```

Comment:
Do we still need this code? If not, why not remove it entirely? We should also use `cy.intercept()` instead of `cy.server()` and `cy.route()`, see [the docs](https://docs.cypress.io/guides/references/migration-guide#cyserver-cyroute-and-CypressServerdefaults).